packages/react/src/components/LinkWithIcon/LinkWithIcon.js

```diff
@@ -20,14 +21,25 @@ const { prefix } = settings;
  * @param {object} props react proptypes
  * @param {object} props.children User content
  * @param {string} props.href Link url
+ * @param {string} props.type Link type
  * @returns {*} LinkWithIcon component
  */
-const LinkWithIcon = ({ children, href, ...props }) => {
+const LinkWithIcon = ({ children, href, type, ...props }) => {
```

Comment:
@RaphaelAmadeu @annawen1 I'm not sure if adding to here makes sense or not? If we do decide to do it this way, we would also need to set the icon by type as well which doesn't seem to be the case here. Perhaps we should only keep this on the `Card` level for now?